tee-worker/scripts/litentry/release/dev_deploy.sh

```diff
@@ -435,34 +455,38 @@ function register_parachain {
 }
 
 function setup_working_dir {
-    echo "Setting up working dir ..."
-    cd "$ROOTDIR/tee-worker/bin" || exit
+  echo "Setting up working dir ..."
+  cd "$ROOTDIR/tee-worker/bin" || exit
 
-    if [ "$PRODUCTION" = false ]; then
-      for f in 'key.txt' 'spid.txt'; do
-        [ -f "$f" ] || touch "$f"
-      done
+  if [ "$PRODUCTION" = false ]; then
+    for f in 'key.txt' 'spid.txt'; do
+      [ -f "$f" ] || touch "$f"
+    done
+  else
+    if [ -e 'key_production.txt' ] && [ -e 'spid_production.txt' ]; then
+      echo "Please make sure 'key_production.txt' and 'spid_production.txt' are existed within $PWD."
     fi
+  fi
 
-    for ((i = 0; i < WORKER_COUNT; i++)); do
-      worker_dir="$WORKER_BASEDIR/w$i"
-      mkdir -p "$worker_dir"
-      for f in 'key.txt' 'spid.txt' 'enclave.signed.so' 'litentry-worker'; do
-        [ -f "$f" ] && cp -f "$f" "$worker_dir"
-      done
+  for ((i = 0; i < $WORKER_COUNT; i++)); do
+    worker_dir="$WORKER_BASEDIR/w$i"
+    mkdir -p "$worker_dir"
+    for f in 'key.txt' 'spid.txt' 'enclave.signed.so' 'litentry-worker'; do
```

Comment:
Same as above, if there are more than two, put them into an array for iteration.